src/pyproject.toml

```diff
@@ -0,0 +1,9 @@
+[tool.black]
+line-length = 100
+# Adjust to the lowest Python version we need to support
+target-version = ["py38"]
```

Comment:
This is something that most repos will need/want to customize. Should we flag it with a specific annotation like "TODO" or "CUSTOMIZE"?

We'll also have to set the base Python version for mypy too.

Reply:
I'm not advocating for that but: jinja2 templates and per-repo configuration?

Reply:
That is kind of our long-term goal (see https://github.com/freedomofpress/securedrop-engineering/issues/15), but for now we're starting off with manual application.

Reply:
A [do-nothing script]!  I like that idea as a way of testing our assumptions about this template repository.  But I'd consider it an enhancement on top of this pull request, not a requirement for it.

[do-nothing script]: https://github.com/freedomofpress/securedrop/pull/6753

Reply:
Yes, that. 🙈 Do-nothing script!

I also agree that's an enhancement, I didn't mean it for this PR.